human-formatter/index.js

```diff
@@ -154,35 +157,38 @@ function prettyStackTrace (c, stack, basepath) {
       let func = match[1]
       let relative = match[2].slice(basepath.length)
       let converted = `${ PADDING }at ${ func } (./${ relative })`
+      // TODO: fix typo
       let isDependecy = match[2].includes('node_modules')
       return isDependecy ? c.gray(converted) : c.red(converted)
     }
   }).join(NEXT_LINE)
 }
 
-class HumanFormatter extends stream.Writable {
-  constructor (options) {
-    super()
+function humanFormatterFactory (options) {
+  let c
+  if (typeof options.color === 'undefined') {
+    c = chalk
+  } else {
+    c = new chalk.Instance({ level: options.color ? 3 : 0 })
+  }
 
-    if (typeof options.color === 'undefined') {
-      this.chalk = chalk
-    } else {
-      this.chalk = new chalk.Instance({ level: options.color ? 3 : 0 })
-    }
+  let basepath = options.basepath || process.cwd()
 
-    this.basepath = options.basepath || process.cwd()
-    this.out = options.out || process.stdout
+  if (basepath.slice(-1) !== path.sep) basepath += path.sep
 
-    if (this.basepath.slice(-1) !== path.sep) this.basepath += path.sep
-  }
+  return function humanFormatter (inputData) {
+    let record
+    if (typeof inputData === 'string') {
+      record = JSON.parse(inputData)
+    } else if (isObject(inputData)) {
+      record = inputData
+    }
+    if (!record) return inputData
```

Comment:
Adapted snippet from official docs - http://getpino.io/#/docs/pretty .
They are also check whether logs are pinos, using `v` property, but it seems that it's removed.